src/library/returnBook/ReturnBookUI.java

```diff
@@ -1,101 +1,100 @@
 package library.returnBook;
+
 import java.util.Scanner;
 
 
 public class ReturnBookUI {
 
-    private enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
+    private enum UiState {INITIALISED, READY, INSPECTING, COMPLETED};
+
+    private ReturnItemControl control;
+
+    private Scanner input;
```

Comment:
#PR42 - D3: Please rename Scanner variable to 'scanner' as per class diagram

Reply:
https://github.com/elliotmitchum/ITC205/pull/42/commits/6e206a82e1b6db14ba97f81a1c51dd5b4773c823